run_mae_pretraining.py  数据没处理好
